include/mbedtls/build_info.h

```diff
@@ -59,6 +59,13 @@
 #define inline __inline
 #endif
 
+/* Define `asm` for compilers which don't define it. */
+/* *INDENT-OFF* */
+#ifndef asm
+#define asm __asm__
+#endif
+/* *INDENT-ON* */
```

Comment:
why do we need the `*INDENT-OFF*` and `*INDENT-ON*`?

Reply:
Because that particular construct confuses uncrustify's parser. Cf changelog message that introduced it originally.

Reply:
Wow, that's a pretty bad parser, as this is not an uncommon construct :( - and also, didn't appear to be necessary in `library/bn_mul.h` (i.e. it wasn't done there)